pkgs/oci/inferenceql.loom/default.nix

```diff
@@ -0,0 +1,25 @@
+{ pkgs,
+  nixpkgs,
+  system,
+  ociImgBase,
+}: let
+  # in OCI context, whatever our host platform we want to build same arch but linux
+  systemWithLinux = builtins.replaceStrings [ "darwin" ] [ "linux" ] system;
+  crossPkgsLinux = nixpkgs.legacyPackages.${systemWithLinux};
+  python = crossPkgsLinux.python3;
+
+  scopes = (import ./../../../lib/mkScopes) pkgs;
+
+  loom = scopes.callPy3Package ./../../loom { };
+in pkgs.dockerTools.buildImage {
+  name = "probcomp/inferenceql.loom";
+  tag = systemWithLinux;
+  fromImage = ociImgBase;
```

Comment:
I suggest not using `fromImage` and just building a single layered image for each instead by composing in whatever base config in Nix-land. You'll end up with a smaller image that's easier to reason about.